src/main/java/org/gridsuite/ds/server/service/DynamicSimulationWorkerService.java

```diff
@@ -260,16 +269,29 @@ protected void clean(AbstractResultContext<DynamicSimulationRunContext> resultCo
         super.clean(resultContext);
         // clean working directory
         Path workDir = resultContext.getRunContext().getWorkDir();
-        removeWorkingDirectory(workDir);
+        removeDirectory(workDir);
```

Comment:
This is the workDir of the microservice, not of the binary right ?
This doesn't break the itool debug mode ?

Reply:
YES, it's workDir at Gridsuite level, not Powsybl. It's used to communicate with powsybl-dynawo in order to provide a location for exporting the Dynawo network dump file.